File: DevcampPracticalProjectMocks/js-project-mock/app/controllers/transactionController.js
const express = require('express');
const transactionController = express.Router();

transactionController.get('/pending', (req, res) => {

    const response = require('../mocks/pending-transactions');

    res.json(response);
    res.end();


});

transactionController.get("/confirmed", (req, res) => {

    const response = require('../mocks/confirmed-transactions');

    res.json(response);
    res.end();

});

transactionController.get('/:txHash', (req, res) => {

    const response = require('../mocks/transaction');

    res.json(response);
    res.end();

});

transactionController.post('/send', (req, res) => {

    const response = JSON.stringify({
        "transactionDataHash": "cd8d9a345bb208c6f9b8acd6b8eefefab20c8a"
    });

    res.json(response);
    res.end();

});

module.exports = transactionController;

File: DevcampPracticalProjectMocks/java-project-mock/src/main/java/org/softuni/web/TransactionController.java
package org.softuni.web;

import org.softuni.utils.ResponseEntityBuilder;
import org.springframework.http.ResponseEntity;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.PathVariable;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

import javax.servlet.http.HttpServletResponse;

@Controller
@RequestMapping("/transactions")
public class TransactionController {

    @RequestMapping("/pending")
    public ResponseEntity<String> getPending(){
        return ResponseEntityBuilder.buildJsonResponseEntity("pending-transactions.json");
    }

    @RequestMapping("/confirmed")
    public ResponseEntity<String> getConfirmed(){
        return ResponseEntityBuilder.buildJsonResponseEntity("confirmed-transactions.json");
    }

    @RequestMapping("/{id}")
    public ResponseEntity<String> getById(@PathVariable("id")String id){
        return ResponseEntityBuilder.buildJsonResponseEntity("transaction.json");
    }

    @RequestMapping(value = "/send", method = RequestMethod.POST)
    public ResponseEntity<String> send(HttpServletResponse response){
        return ResponseEntityBuilder.buildJsonResponseEntity("transaction-send.json");
    }

}
